examples/howto/optimize_rhythmic.py

```diff
@@ -0,0 +1,126 @@
+"""
+=========================================
+05. Optimize simulated rhythmic responses
+=========================================
+
+This example demonstrates how to optimize the power spectral density (PSD)
+of a sensory evoked response in the alpha and beta frequency bands.
```

Comment:
```suggestion
of a current dipole signal in the alpha and beta frequency bands.
```
What do you think? Since evoked responses are typically assessed in the time domain

Reply:
How about “of a current dipole signal with significant alpha and beta spectral components”?